### PR TITLE
Add support for options hashes without braces

### PR DESCRIPTION
**Pull request by searls**
Picks up the work of #14 (PR sent b/c I didn't have push access to @calebhearth's fork)

### CODE REVIEW

File: test/unit/imitates_type/makes_double/declares_dry_class_test.rb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      @subject = DeclaresDryClass.new
+    end
+
+    class Fib
+      def lie(truth, lies:)
+      end
+    end
+
+    def test_calls_handle_dry_call_with_what_we_want
+      fake_fib_class = @subject.declare(Fib, [:lie])
```
Comment (calebhearth) [inline test/unit/imitates_type/makes_double/declares_dry_class_test.rb:17]: Oh good you'll get something out of my talk on not using subject, even if you're not using RSpec.

File: test/safe/of_test.rb [hunk 2]
```diff
@@ -27,7 +27,7 @@ def test_neato
 
   class Welp
     def to_s
-      "¯\_(ツ)_/¯"
+      "¯\\_(ツ)_/¯"
```
Comment (calebhearth) [inline test/safe/of_test.rb]: <3

Comment (jasonkarns) [inline test/safe/of_test.rb:30]: y u no 🤷

Comment (searls) [reply test/safe/of_test.rb:30]: Lol this was actually just a new Standard fix

File: test/unit/stringifies_method_signature_test.rb [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+    unless Gem::Version.new(RUBY_VERSION) < Gem::Version.new("3.1")
+      eval(<<~RUBY, binding, __FILE__, __LINE__ + 1) # standard:disable Security/Eval
+        def dotdotdot(...)
+        end
+
+        def dotdotdot_with_args(a, b = :foo, ...)
+        end
+      RUBY
+    end
```
Comment (calebhearth) [inline test/unit/stringifies_method_signature_test.rb:95]:
```suggestion
    unless Gem::Version.new(RUBY_VERSION) < Gem::Version.new("3.1")
      def dotdotdot(...)
      end

      def dotdotdot_with_args(a, b = :foo, ...)
      end
    end
```

You can safely gate definitions behind conditionals, even if they contain (this) invalid syntax
```
 ❯ chruby 2.7
irb(main):001:1* if false
irb(main):002:1*   def dotdotdot(...); end
irb(main):003:0> end
=> nil
irb(main):004:0> 
Traceback (most recent call last):
        4: from /Users/caleb/.rubies/ruby-2.7.5/bin/irb:23:in `<main>'
        3: from /Users/caleb/.rubies/ruby-2.7.5/bin/irb:23:in `load'
        2: from /Users/caleb/.rubies/ruby-2.7.5/lib/ruby/gems/2.7.0/gems/irb-1.2.6/exe/irb:11:in `<top (required)>'
        1: from (irb):7
NameError (undefined local variable or method `dotdotdot' for main:Object)
```

**Review by calebhearth — APPROVED** [review]

File: test/unit/imitates_type/makes_double/declares_dry_class_test.rb (hunk 1 above)
Comment (searls) [reply test/unit/imitates_type/makes_double/declares_dry_class_test.rb:17]: Sorry man I'm a hardcore fan of consistently calling the SUT subject

File: test/safe/kwargs_vs_options_hash_test.rb [hunk 4]
```diff
@@ -0,0 +1,36 @@
+require "test_helper"
+
+class KwargsVsOptionsHashTest < Minitest::Test
```
[the remark this answers is not in the document]
Comment (searls) [reply test/safe/kwargs_vs_options_hash_test.rb:3]: At least you'll always have the git log!

**Comment by calebhearth** [comment]
Feelsgoodman

**Pull request merged**